docs/workflows/common-workflow-steps/sauce-labs-saucectl.md

```diff
@@ -66,8 +57,24 @@ artifacts:
     when: always
     directory: artifacts
 ```
-And sample `config.yml` file for [Xcuitest](https://docs.saucelabs.com/mobile-apps/automated-testing/espresso-xcuitest/xcuitest/):
-```yml title="/.sauce/config.yml"
+
+:::
+
+#### For iOS
+
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Xcodebuild for Testing**](/workflows/ios-specific-workflow-steps#xcodebuild-for-testing) | After the [**Xcodebuild for Testing**](/workflows/ios-specific-workflow-steps#xcodebuild-for-testing) step runs, the test IPA and APP paths (`$AC_TEST_IPA_PATH` and `$AC_UITESTS_RUNNER_PATH`) will be created automatically. So that the **Saucectl Run** component depends on these paths. |
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE4255-saucectlRunFlow.png' />
```

Comment:
This SS has some problems:
- Component name is not up-to-date
- It seems to be a **Custom Script** instead of native step (!)